FASTN.ftd

```diff
@@ -662,6 +662,8 @@ skip: true
     document: best-practices/self-referencing.ftd
   - Property: /property-guidelines/
     document: best-practices/property-guidelines.ftd
+  - Enviroment Variables: /best-practices/env-vars
```

Comment:
URLs must end with slash. Also no need to use document if the URL and document names are matching. Also this is not a best practice, this is a proper feature. We have to document it in the fastn.com -> Docs -> FullStack section of our site.